### PR TITLE
Following the license change, modify the boilerplates in last few

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: e_os.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
  *
- * Licensed under the OpenSSL license (the "License").  You may not use
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
```
Comment (mattcaswell) [inline e_os.h]: Typo here. Should be Licence. Would you mind updating here and all the other similar places in the other PRs?

**Review by mattcaswell — CHANGES_REQUESTED** [review]

**Review by mattcaswell — APPROVED** [review]
Just kidding... :-)

**Comment by levitte** [comment]
Fortunately, I saw "Just kidding" first...

**Comment by levitte** [comment]
Merged.

402dd5585e1e23f035f86e9c2f3c26341b4a42bf Following the license change, modify the boilerplates in last few

**Pull request closed without merging**